src/utils/__tests__/actions.spec.js

```diff
@@ -18,6 +22,12 @@ describe('actions', () => {
       bookmark = 'someBookmark';
       variable = 'someVariable';
       qStateName = 'someState';
+      automation = 'someAutomation';
+      executePath = 'someAutomationExecutionPath';
+      automationPostData = false;
+      executeAutomation = {
+        fetch: sinon.spy(),
+      };
```

Comment:
This is not the correct way to mock fetch. Let's wait with fixing this until we are sure on what to use for the REST calls

Reply:
switched to Axios